heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -98,23 +102,33 @@ local msg = {
     Logger      = "fx",
     Type        = "executive_summary",
     Fields      = {
-        {name = "clientId"  , value = ""},
-        {name = "documentId", value = ""},
-        {name = "geo"       , value = ""},
-        {name = "channel"   , value = ""},
-        {name = "os"        , value = ""},
-        {name = "hours"     , value = 0},
-        {name = "crashes"   , value = 0, value_type = 2},
-        {name = "default"   , value = false},
-        {name = "google"    , value = 0, value_type = 2},
-        {name = "bing"      , value = 0, value_type = 2},
-        {name = "yahoo"     , value = 0, value_type = 2},
-        {name = "other"     , value = 0, value_type = 2},
+        {name = "clientId"          , value = ""},
+        {name = "documentId"        , value = ""},
+        {name = "geo"               , value = ""},
+        {name = "channel"           , value = ""},
+        {name = "os"                , value = ""},
+        {name = "hours"             , value = 0},
+        {name = "crashes"           , value = 0, value_type = 2},
+        {name = "default"           , value = false},
+        {name = "google"            , value = 0, value_type = 2},
+        {name = "bing"              , value = 0, value_type = 2},
+        {name = "yahoo"             , value = 0, value_type = 2},
+        {name = "other"             , value = 0, value_type = 2},
+        {name = "reason"            , value = ""},
+        {name = "sessionId"         , value = ""},
+        {name = "subsessionCounter" , value = 0, value_type = 2},
+        {name = "buildId"           , value = ""},
+        {name = "hangs"             , value = 0, value_type = 2},
```

Comment:
we should probably change the name to `pluginHangs` or similar then. It may be confusing with chrome hangs / browser hangs etc.